tests/CMakeLists.txt

```diff
@@ -259,9 +263,9 @@ endif()
 #
 message(STATUS "Using C++11")
   if (NOT CMAKE_COMPILER_IS_GNUCXX OR GCC_VERSION VERSION_GREATER 4.6)
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=gnu++11")
```

Comment:
Ah, qmake might insert something there which overrides our attempt to specify `-std=c++11`.
Let's leave it like you suggested.
Could you add a comment in the CMakeFiel that gnu++11 is required to get math constants defined, so we remember this in the future?